src/typeahead/typeahead.ts

```diff
@@ -284,13 +288,20 @@ export class NgbTypeahead implements ControlValueAccessor,
       this._windowRef.instance.selectEvent.subscribe((result: any) => this._selectResultClosePopup(result));
       this._windowRef.instance.activeChangeEvent.subscribe((activeId: string) => this.activeDescendant = activeId);
 
+      const windowElement = this._windowRef.location.nativeElement;
+      this._removeMousedownListener =
+          this._renderer.listen(windowElement, 'mousedown', (event) => { event.preventDefault(); });
```

Comment:
@benouat has just suggested to remove all this renderers and listeners and move this all to the `typeahead-window.ts`... it would be just a one-liner, no idea how we missed it before :)

```ts
@Component({
  selector: 'ngb-typeahead-window',
  exportAs: 'ngbTypeaheadWindow',
  host: {
    '(mousedown)': '$event.preventDefault()',
    /* ... */
  },
```